packages/ai-jsx/package.json

```diff
@@ -180,6 +180,16 @@
         "default": "./dist/cjs/lib/openai.cjs"
       }
     },
+    "./lib/anthropic": {
```

Comment:
Dumb question out of curiosity: Why is it necessary to do this for each new source file in the tree? This seems kinda weird to me...

Reply:
It's not every new source file; it's just the source files which are exported. Which is a lot right now. 🙃 

If we moved to a model where we had separate npm packages (e.g. `npm install ai-jsx @ai-jsx/tools @ai-jsx/router`), then we'd have less need to do this. But the monopackage pushes us more in this direction.

I agree it's not great and in the future we may change it.

LC does something similar fwiw https://github.com/hwchase17/langchainjs/blob/main/langchain/package.json.